pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, python35Packages
+, stdenv
+, bash
+, intltool
+, glib
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, wrapGAppsHook
+}:
+
+python35Packages.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
```

Comment:
The Python libraries should be in `propagatedBuildInputs`. Executables that you need during runtime need to be added to `PATH`. Libraries you need during runtime need to be added to `LD_LIBRARY_PATH`, or preferably hardcoded in your package.